test/apiClient.test.ts

```diff
@@ -54,3 +64,13 @@ describe('API Client Integration Tests', () => {
     await api.workspaces.deleteWorkspace(workspace);
   });
 });
+
+async function getWorkspace(id: string): Promise<Schemas.Workspace> {
+  try {
+    const result = await api.workspaces.getWorkspace(id);
+    return result;
+  } catch (error) {
+    await new Promise((resolve) => setTimeout(resolve, 2000));
+    return getWorkspace(id);
+  }
+}
```

Comment:
This is to avoid 401 while API key update replication is happening across regions